projects.yaml

```
@@ -520,3 +520,8 @@ projects:
      category: ml
      homepage: https://www.mathworks.com/products/reinforcement-learning.html
      license: Proprietary Software License
+   - name: BRAX
+     github_id: https://github.com/google/brax
```

```suggestion
     github_id: google/brax
```

It only inputs the organization name and the project name